compose/monitoring.yml

```diff
@@ -44,6 +52,55 @@ services:
       db:
         condition: service_started
     restart: unless-stopped
+  sql-exporter:
+    image: burningalchemist/sql_exporter
+    container_name: sql-exporter
+    expose:
+      - 9399
+    command: --config.file=/etc/sql-exporter/sql-exporter.yml
+    volumes:
+      - ./default/sql-exporter.yml:/etc/sql-exporter/sql-exporter.yml
+    depends_on:
+      prometheus:
+        condition: service_started
+      db:
+        condition: service_started
+    restart: unless-stopped
+
+  solr-exporter:
+    build:
+      context: build/solr
+      args:
+        - MB_SOLR_VERSION=${MB_SOLR_VERSION:-solr-9.7.0}
+    image: musicbrainz-docker_search:${MB_SOLR_VERSION:-solr-9.7.0}
+    container_name: solr-exporter
+    expose:
+      - 9854
+    command: sh -c 'cd /opt/solr-9.7.0/contrib/prometheus-exporter; exec bin/solr-exporter -p 9854 -z search:9983 -f conf/solr-exporter-config.xml'
+    depends_on:
+      prometheus:
+        condition: service_started
+      search:
+        condition: service_healthy
+    restart: unless-stopped
+
+  node-exporter:
+    image: prom/node-exporter:latest
+    container_name: node-exporter
+    restart: unless-stopped
+    expose:
+      - 9100
+    volumes:
+      - "/:/host:ro,rslave"
+      - node-exporter-textfile-collector:/textfile-collector-directory
+    command:
+      - --path.rootfs=/host
+      - --collector.filesystem.ignored-fs-types="^(autofs|binfmt_misc|cgroup|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|mqueue|nsfs|tmpfs|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|sysfs|tracefs)$"
```

Comment:
FYI, not sure if it applies to the version we run yet, but:

According to https://github.com/prometheus/node_exporter/blob/810510e12b063690e6e52700a867676b93492e92/collector/filesystem_common.go#L262

"--collector.filesystem.ignored-fs-types is DEPRECATED and will be removed in 2.0.0, use --collector.filesystem.fs-types-exclude"

And according to https://github.com/prometheus/node_exporter/blob/810510e12b063690e6e52700a867676b93492e92/collector/filesystem_common.go#L233

"--collector.filesystem.ignored-mount-points is DEPRECATED and will be removed in 2.0.0, use --collector.filesystem.mount-points-exclude"

I checked the code, and I don't see a good reason for it to duplicate entries but if they are read twice (any symlink or multiple mounts?) plus it *should* be ignored.